functions/public/Invoke-WPFShortcut.ps1

```diff
@@ -27,9 +27,19 @@ function Invoke-WPFShortcut {
             $ArgumentsToSourceExe = "$powershell '$IRM'"
             $DestinationName = "WinUtil.lnk"
 
+            Invoke-WebRequest -Uri "https://christitus.com/images/logo-full.png" -OutFile "$env:TEMP\cttlogo.png"
+
             if (Test-Path -Path "$env:TEMP\cttlogo.png") {
-                $iconPath = "$env:SystempRoot\cttlogo.ico"
-                ConvertTo-Icon -bitmapPath "$env:TEMP\cttlogo.png" -iconPath $iconPath
+                $iconPath = "$env:TEMP\cttlogo.ico"
```

Comment:
I would still rather put this into a persistent location like "$env:LOCALAPPDATA\winutil" to make sure the shortcut never loses its icon even if the user doesn't use it for a while